src/main/java/mate/academy/bookstore/dto/CartItemRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.bookstore.dto;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.Data;
+
+@Data
+public class CartItemRequestDto {
+    @NotNull
+    @Positive
+    private Long bookId;
+    @Positive
+    private int quantity;
```

Comment:
what will happen if a user sends null here?

Reply:
I'll receive an error in response 
```
"errors": [
        {
            "field": "cartItems[].quantity",
            "message": "must be greater than 0"
        }
    ]
```